lzy/slots/src/main/java/ai/lzy/fs/backends/OutputPipeBackend.java

```diff
@@ -0,0 +1,66 @@
+package ai.lzy.fs.backends;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.*;
+import java.nio.channels.Channels;
+import java.nio.channels.FileChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class OutputPipeBackend implements OutputSlotBackend {
+    public static final Logger LOG = LogManager.getLogger(OutputPipeBackend.class);
+
+    private final Path pipePath;
+    private final File storageFile;
+    private final AtomicBoolean isReady = new AtomicBoolean(false);
+
+    public OutputPipeBackend(Path pipePath) throws IOException, InterruptedException {
+        this.pipePath = pipePath;
+        storageFile = File.createTempFile("storage", ".tmp");
+
+        if (!pipePath.getParent().toFile().exists()) {
+            Files.createDirectories(pipePath.getParent());
+        }
+
+        // Creating named pipe
+        var res = Runtime.getRuntime().exec(new String[]{"mkfifo", pipePath.toAbsolutePath().toString()}).waitFor();
```

Comment:
what about Windows? :)

Reply:
I think we cannot support workers on windows))